src/main/resources/application-migration.yml

```diff
@@ -0,0 +1,29 @@
+localstack:
+  base-url: http://${localstack.config.host}:${localstack.config.port}
+  config:
+    host: localhost
+    port: 4566
+    region: eu-west-2
+
+
+aws:
```

Comment:
As you should be using te main queue here - this isn't necessary.

Reply:
Fixed